index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./CSS/style.css">
+</head>
+<body>
+  <div class="container">
+    <!-- Header  -->
+    <header>
+       <nav class="top-nav">
+            <h1 class="logo">My log</h1>
+          <ul>
+              <li>
+                <img class="vector" src="./images/Img/Union.png" alt="Union">
+              </li>
+          </ul> 
+       </nav>
+       <!-- /images/Img/Union.png -->
+<!-- header inline section -->
+        <section >
+              <h2 class="greeting">
+                I'm Michael.
+                Glad to see you!
+              </h2>
+        
+              <article>
+                  <p class="intro">
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project 
+                    you need coded, don’t hesitate to contact me.
+                  </p>
+              </article>
+        </section>
+
+        <!-- header footer section -->
+        <section>
+                <div>
+                  <a class="network" href="#"><h4>LET'S CONNECT!</h4></a>
```

Comment:
-[OPTIONAL] Kindly make sure to use h4 only if you already have h3 in order to meet the semantic HTML rules :recycle: